source/P0323R9.bs

```diff
@@ -1238,11 +1243,11 @@ template<class T1, class E1, class T2> constexpr bool operator!=(const expected<
 template<class T1, class E1, class T2> constexpr bool operator!=(const T2& v, const expected<T1, E1>& x);
 ```
 
-*Mandates*: `T1` and `T2` are not *cv* `void` and the expression `*x == v` is well-formed
+*Mandates*: `T1` and `T2` are not *cv* `void` and the expression `*x != v` is well-formed
 and its result is convertible to
 `bool`. [ *Note:* `T1` need not be *EqualityComparable*. - *end note*]
 
-*Returns*: `bool(x) ? *x != v : false;`.
+*Returns*: `bool(x) ? *x != v : true;`.
```

Comment:
If `x` holds an error, then is it unequal to **any** value `v`. Otherwise, `!( x != v)` would not be the same as `x == v`.